discord/member.py

```diff
@@ -398,7 +398,7 @@ def display_name(self):
         if they have a guild specific nickname then that
         is returned instead.
         """
-        return self.nick if self.nick is not None else self.name
+        return self.nick or self.name
```

Comment:
Technically these aren't equivalent, if e.g. `self.nick` is an empty string. As far as I know though this isn't possible at the API layer so it should be fine.